lib/plugins/extension/lang/en/intro_search.txt

```diff
@@ -1 +1 @@
-This tab gives you access to all available 3rd party plugins and templates for DokuWiki. Please be aware that installing 3rd party code may pose a **security risk**, you may want to read about [[doku>security#plugin_security|plugin security]] first.
\ No newline at end of file
+This tab gives you access to all available 3rd party [[doku>plugins]] and [[doku>template]]s for DokuWiki. Please be aware that installing 3rd party code may pose a **security risk**, you may want to read about [[doku>security#plugin_security|plugin security]] first.
```

Comment:
```suggestion
This tab gives you access to all available 3rd party [[doku>plugins|plugins]] and [[doku>template|templates]] for DokuWiki. Please be aware that installing 3rd party code may pose a **security risk**, you may want to read about [[doku>security#plugin_security|plugin security]] first.
```

Reply:
Thank you. Indeed, I can translate to farsi, zhou etc.! I feel all those countries and people so closer... :) Updated.